src/ax/saga/eventsourcing/messagestore.go

```diff
@@ -0,0 +1,84 @@
+package eventsourcing
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/jmalloc/ax/src/ax"
+	"github.com/jmalloc/ax/src/ax/persistence"
+	"github.com/jmalloc/ax/src/ax/saga"
+)
+
+// streamName returns the message store stream name that contains the events for
+// the given instance.
+func streamName(id saga.InstanceID) string {
+	return "saga:" + id.Get()
+}
+
+// appendEvents appends the events in envs to the message stream for the given
```

Comment:
This seems ok, but should it be talking about the "message stream" or "message store".

Reply:
This one is ok, internally it calls `ms.AppendEvents()`, the docs of which are:

```go
// AppendMessages appends one or more messages to a named stream.
```